PWGJE/EMCALJetTasks/macros/JetQA/scalePtHardHistos.py

```diff
@@ -353,15 +440,38 @@ def MovingAverage(hist, index, numberOfCountsBelowIndex = 0, numberOfCountsAbove
 ########################################################################################################
 # Plot basic histogram    ##############################################################################
 ########################################################################################################
-def plotHist(h, outputFilename, drawOptions = "", setLogy = False):
+def plotHist(h,hAfter, pTHardBin, outputFilename, drawOptions = "", setLogy = False):
```

Comment:
I would suggest to change the name of this function to something more descriptive, since it now serves a specific purpose.

Reply:
renamed.